shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/repository/AuthRepositoryImpl.kt

```diff
@@ -0,0 +1,37 @@
+package co.nimblehq.mark.kmmic.data.repository
+
+import co.nimblehq.mark.kmmic.data.service.auth.AuthService
+import co.nimblehq.mark.kmmic.data.service.auth.model.AuthLoginParams
+import co.nimblehq.mark.kmmic.domain.model.AuthToken
+import co.nimblehq.mark.kmmic.domain.repository.AuthRepository
+import com.russhwolf.settings.ExperimentalSettingsApi
+import com.russhwolf.settings.Settings
+import com.russhwolf.settings.serialization.decodeValueOrNull
+import com.russhwolf.settings.serialization.encodeValue
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.serialization.ExperimentalSerializationApi
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+private const val ACCESS_TOKEN_KEY = "access_token"
```

Comment:
For Kotlin, it's prefered to put `KEY` in the first place
```suggestion
private const val KEY_ACCESS_TOKEN = "access_token"
```

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)